### ISSUE TITLE
Report to identify records marked public w/o rights statement

### ISSUE THREAD

**Issue by HKativa** [sciencehistory/chf-sufia#1069]
Reminder: DPLA requires a rights statement; depending on when this ticket is addressed, we may have to omit such records from our OAI-PMH feed, and/or let PA Digital omit them from what it sends to DPLA.

**Comment by eddierubeiz** [comment]
Good news! Based on a cursory check, it appears that all public works have rights statements at the moment. (I double-checked with Cat, and she confirmed that the one problematic item I was able to find in Staging had in fact been flagged by the DPLA folks at Temple and corrected in the mean time in Production.)

Should we want to run this cursory check again periodically, I've added the recipe to the wiki at: https://chemheritage.atlassian.net/wiki/spaces/HDCSD/pages/15368194/Application+administration .

@HKativa , @catlu , @jrochkind , if we want to turn this code into a permanently available page in the admin section, I can of course do that, but it seems like overkill at the moment. Your thoughts are welcome.
